### DISCUSSION TITLE
Are you open for design contributions?

### DISCUSSION THREAD

**Discussion by fivenp** [GyulyVGC/sniffnet#38] [General]
Hi there. 

I really fell in love with the functionality and simplicity of sniffnet and got curious whether you'd be open for designers to contribute some thoughts around UI/UX and visual style to it? :) 

If so I'd love to help you set up some basic design contribution guidelines and bring it in front of talented designers to help you evolve it.

**Comment by GyulyVGC** [comment]
Hi! 

Thank you very much for this proposal, it would be amazing.

The only thing is that I developed Sniffnet using [iced](https://github.com/iced-rs/iced) which is a not-so-mature GUI library, like many other GUI library written in Rust, so I have some light limitations.

However I'm open to opinions and suggestions, especially if they come from more expert people!
 I was also considering adding more themes other than the night/day ones, so it would be a nice occasion for me to get some hints!

**Comment by fivenp** [comment]
Amazing.... I think iced shouldn't be a real blocker here – it seems to be flexible enough to allow a lot of customisation :) 

I'm currently launching https://contribute.design which intends to give you guidance how to setup the guidelines (and would list sniffnet) 

Do you want to try setting it up yourself (and include me in the PR) shall I propose a draft in a PR? :)

**Comment by GyulyVGC** [comment]
I can try by myself, I will let you know!
